v2.0/alter-sequence.md

```diff
@@ -34,11 +35,50 @@ table td:first-child {
 
  Parameter | Description
 -----------|------------
+`IF EXISTS` | Modify the sequence only if it exists; if it does not exist, do not return an error.
+`sequence_name` | The name of the sequence you want to modify.
+`INCREMENT` | The new value by which the sequence is incremented. A negative number creates a descending sequence. A positive number creates an ascending sequence.
+`MINVALUE` | The new minimum value of the sequence.
```

Comment:
You might want to specify the default values for `MINVALUE` and `MAXVALUE`

Reply:
added